examples/dqn/dqn.py

```diff
@@ -3,75 +3,66 @@
 # This source code is licensed under the MIT license found in the
 # LICENSE file in the root directory of this source tree.
 
+import dataclasses
```

Comment:
I have a meta question here (for @jieru-hu I think): Do we have any preference between using dataclasses vs attrs for making configs (from Hydra's perspective) ? I think not but would like to double check.

Reply:
no preference from OmegaConf's standpoint (we support and test both).
`dataclass` seems to be more commonly used though.

Reply:
Cool - lets stick to dataclasses then :)